packages.dhall

```diff
@@ -1,9 +1,64 @@
-let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.15.4-20220725/packages.dhall
-        sha256:e56fbdf33a5afd2a610c81f8b940b413a638931edb41532164e641bb2a9ec29c
+let mkPackage =
+  https://raw.githubusercontent.com/purescript/package-sets/psc-0.13.0-20190626/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57
 
+let upstream =
+  https://github.com/purescript/package-sets/releases/download/psc-0.15.4-20221201/packages.dhall
+    sha256:d1a68fa15709eaa686515eb5b9950d82c743f7bf73e3d87a4abe9e1be6fda571
 let overrides = {=}
 
+let additional_dependencies = upstream
```

Comment:
This list is formatted pretty inconsistently, is there a dhall formatter (on at least this one off basis) we could use?

Reply:
I think it's also worth leaving a note about what's going on here (using a different upstream for these dependencies) as it's a bit odd at first glance. Even if just something like `This is copied over from Foo dependency`

Reply:
I was going to ask about that too. Is this just a consequence of needing to pull in additional dependencies to support the new version of         , repo = "https://github.com/rightfold/purescript-postgresql-client.git" ?